lib/forms.js

```diff
@@ -72,6 +72,18 @@ exports.create = function (fields, options) {
             return b;
         },
         handle: function (obj, callbacks) {
+            var returnValue = undefined;
+
+            if (!callbacks) {
+                returnValue = new Promise(resolve => {
+                    callbacks = {
+                        success: form => resolve({ success: true, form }),
+                        error: form => resolve({ error: true, form }),
+                        empty: form => resolve({ empty: true, form }),
```

Comment:
```suggestion
                returnValue = new Promise(function (resolve) {
                    callbacks = {
                        success: function (form) { return resolve({ success: true, form }); },
                        error: function (form) { return resolve({ error: true, form }); },
                        empty: function (form) { return resolve({ empty: true, form }); },
```

Reply:
I at times forget how verbose Promises and callbacks with non-arrow functions are 😝